Tasks/AndroidSigningV3/postandroidsigning.ts

```diff
@@ -0,0 +1,21 @@
+import fs = require('fs');
```

Comment:
switch to import * as fs syntax?

Reply:
(modules imported with import-from are treated as readonly, while import-require basically gives you a JS object)